doc(v2): describe better the disks supported (#953)
longhorn/website: content/docs/1.7.0/v2-data-engine/prerequisites.md

### Disk

> SPDK can use every kind of disk that is supported by Linux, making leverage on Kernel drivers. But SPDK is equipped with a user space NVMe driver which provide zero copy, highly parallel access directly to an SSD from an user space application, so 
>  **Local NVMe disks** are highly recommended for optimal storage performance of volumes using V2 Data Engine.

Reviewer: ```suggestion
SPDK leverages kernel drivers to support every kind of disk that Linux supports. However, SPDK is equipped with a user space NVMe driver that provides zero-copy, highly parallel, direct access to an SSD from a user space application. Because of this, using **local NVMe disks** is highly recommended for enabling V2 volumes to achieve optimal storage performance.
```

Author: Much better, thanks.
